Header header
uint8 direction
bool enable_hold
uint32 steps_desired
